mainmatter/introduction/pipeline/index.tex

```diff
@@ -0,0 +1,96 @@
+\section{The Graphics Pipeline} \label{sec:pipeline}
+\begin{sectionmeta}
+To understand the following sections it is recommended to gather an understanding of what the graphics pipeline is.
```

Comment:
Recommend is odd, can that be changed please? (necessary)

Reply:
Ikke enig